Failed to execute goal org.apache.maven.plugins:maven-changes-plugin:2.12:announcement-generate

[WARNING] The requested profile "maxur-ldoc" could not be activated because it does not exist.
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-changes-plugin:2.12:announcement-generate (include-announcement-file) on project maxur-ldoc: Something is wrong with the Issue Management section. See previous error messages. -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-changes-plugin:2.12:announcement-generate (include-announcement-file) on project maxur-ldoc: Something is wrong with the Issue Management section. See previous error messages.
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
    at io.takari.maven.builder.smart.SmartBuilderImpl.buildProject(SmartBuilderImpl.java:334)
    at io.takari.maven.builder.smart.SmartBuilderImpl$ProjectBuildTask.run(SmartBuilderImpl.java:104)
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
    at java.util.concurrent.FutureTask.run(FutureTask.java:266)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:745)
Caused by: org.apache.maven.plugin.MojoExecutionException: Something is wrong with the Issue Management section. See previous error messages.
    at org.apache.maven.plugin.announcement.AnnouncementMojo.execute(AnnouncementMojo.java:566)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:207)
    ... 10 more

--- a/.rultor.yml
+++ b/.rultor.yml
@@ -17,4 +17,4 @@ release:
   script: |
     mvn versions:set "-DnewVersion=${tag}" -Denv=prod
     git commit -am "${tag}"
-    mvn clean deploy --settings /home/r/settings.xml -Denv=prod -Pmaxur-ldoc,sonatype,debug
+    mvn clean deploy --settings /home/r/settings.xml -Denv=prod -Pmaxur,sonatype,debug,-announcement

